detect_variants/tumor_only_detect_variants.cwl

```diff
@@ -32,15 +32,17 @@ inputs:
         default: 2
     maximum_population_allele_frequency:
         type: float?
-        default: 0.001
+        default: 0.1
     vep_cache_dir:
         type: string?
     synonyms_file:
         type: File?
     coding_only:
         type: boolean?
+        default: true
     hgvs_annotation:
         type: boolean?
+        default: true
     variants_to_table_fields:
         type: string[]?
         default: [CHROM,POS,ID,REF,ALT,set]
```

Comment:
While we are at it, I think we can remove `ID`. It's not informative in our annotated VCFs.

Reply:
Done